doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -451,3 +454,155 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
    usage::
 
       --ssram
+
+.. _emul_config:
+
+Emulated PCI Device Types
+****************************
+
+In the acrn-dm ``-s`` or ``--pci_slot`` command line parameter, there is a ``<slot_config`` argument
+that contains a string describing the type of emulated PCI device along with optional device-dependent
+arguments used for configuration.  Here is a table describing these emulated device types:
+
+.. list-table:: Emulated PCI Device Types
+   :header-rows: 1
+
+   * - PCI Device Type string 
+     - Description
+
+   * - "xhci"
+     - USB controller used to support USB3.0 devices, also supports USB2.0 and USB1.0 devices.
+       Parameter <bus number>-<port number> should be added, physical USB device attached on the bus
+       and port will be detected by User VM and be used as expected, e.g.xhci,1-2,1-2.
+
+   * - "lpc"
+     - Low Pin Count (LPC) bus is used to connect low speed devices to the cpu, for example serial port,
+       keyboard, or mouse.
+
+   * - "igd-lpc"
+     - Windows graphic driver requires this virtualized LPC device to complete the display function.
+
+   * - "ivshmem"
+     - Inter-VM shared memory (ivshmem) virtualized PCI device used specifically for shared
+       memory between VMs. Parameters should be added with the format ivshmem,<shm_name>,<shm_size>,
+       <shm-name> specifies a shared memory name, this shm_name must be listed in hv.FEATURES.IVSHMEM.IVSHMEM_REGION
+       configured in config tool UI, and needs to start with dm:/ prefix. <shm_size> specifies shared memory size of
+       selected shm_name.
+
+   * - "ahci"
+     - Advanced Host Controller Interface provides advanced features to access Serial ATA (SATA)
+       storage devices, such as a hard disk. Parameter <type:><filepath>* should be added, type could be 'hd'(harddisk)
+       and 'cd'(CD-ROM). <filepath> is the path for the backend file, could be a partition or a regular file,
+       e.g. ahci,hd:/dev/sda.
+
+   * - "ahci-hd"
+     - It is an alias for "ahci"
+
+   * - "ahci-cd"
+     - Advanced Host Controller Interface used to connect with AT Attachment
+       Packet Interface device (for CD-ROM emulation). "ahci-cd" has the same parameters with "ahci".
+
+   * - "amd_hostbridge"
+     - Virtualized PCI AMD hostbridge
+
+   * - "hostbridge"
+     - Virtualized PCI hostbridge, a hardware bridge between the CPU's high-speed system local bus
+       and the Peripheral Component Interconnect (PCI) bus.
+
+   * - "pci_gvt"
+     - Intel GVT-g technology provides mediated device passthrough for Intel GPUs (Broadwell and newer).
+       It can be used to virtualize the GPU for multiple guest VMs, providing near-native graphics performance.
+
+   * - "virtio-blk"
+     - Virtio block type device, a string could be appended to specify the disk image,e.g.virtio-blk,/root/test.img,
+       user also could use "nodisk" to create a virtio-blk device with dummy backend.
```

Comment:
What does that exactly do when you use a dummy backend? What do you see from the User VM; empty disk, just a controller, etc.?